2018_Fall/readme.md

```diff
@@ -31,7 +31,7 @@ Make the Pull Request from your own GitHub account. Make sure you're changing th
 - [ ] Cornejo, Ricky
 - [ ] Crestani, Ryan
 - [ ] Czymoch, Maximilian
-- [ ] D'Agostino, Alex
+- [X] D'Agostino, Alex:pinneaple::boom:
```

Comment:
I think it's `:pineapple:`

Reply:
Thanks Joe! I think I edited it - let me know how it looks now!